cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -155,7 +155,7 @@ func (r *KubeArchiveConfigReconciler) desiredRole(kaconfig *kubearchivev1alpha1.
 		if slash > -1 {
 			apiGroup = resource.APIVersion[:slash]
 		}
-		rules = append(rules, rbacv1.PolicyRule{APIGroups: []string{apiGroup}, Resources: []string{strings.ToLower(resource.Kind)}, Verbs: []string{"get", "list", "watch"}})
+		rules = append(rules, rbacv1.PolicyRule{APIGroups: []string{apiGroup}, Resources: []string{strings.ToLower(resource.Kind) + "s"}, Verbs: []string{"get", "list", "watch"}})
```

Comment:
Does the go formatter support wrapped lines?

```suggestion
		rules = append(rules, rbacv1.PolicyRule{
		        APIGroups: []string{apiGroup},
		        Resources: []string{strings.ToLower(resource.Kind) + "s"},
		        Verbs: []string{"get", "list", "watch"}})
```

Reply:
This is temporary, but I want to get it checked in so things will work for people. I'll open another issue to fix this the right way.